frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -103,7 +103,7 @@ export const installPlanForSubscription = (
   installPlans: InstallPlanKind[] = [],
   subscription: SubscriptionKind,
 ): InstallPlanKind =>
-  installPlans.find((ip) => ip?.metadata?.name === subscription?.status?.installPlan?.name);
```

Comment:
Unfortunately, TypeScript didn't catch this since we used `& K8sResourceKind` in the type definition. We should try to move to `& K8sResourceCommon` for our types, which has more strict checking.